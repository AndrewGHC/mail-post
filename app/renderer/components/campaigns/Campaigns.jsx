import React, { Fragment } from 'react';

const Campaigns = () => (
  <Fragment>
    Campaigns
  </Fragment>
);

export default Campaigns;
