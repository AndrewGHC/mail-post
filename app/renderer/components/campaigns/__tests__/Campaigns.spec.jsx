import React from 'react';
import { shallow } from 'enzyme';

import Campaigns from '../Campaigns';

describe('Campaigns', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<Campaigns />);
    expect(wrapper).toMatchSnapshot();
  });
});
