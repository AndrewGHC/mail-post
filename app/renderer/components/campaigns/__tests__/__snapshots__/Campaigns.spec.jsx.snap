// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Campaigns matches snapshot 1`] = `
<React.Fragment>
  Campaigns
</React.Fragment>
`;
